portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase

```diff
@@ -671,6 +671,78 @@ definition {
 			value1 = "test@liferay.com");
 	}
 
+	@description = "This is a use case for LPS-105156."
+	test RemoveExpiredAuthorization {
+		property osgi.module.configuration.file.names = "com.liferay.oauth2.provider.rest.internal.spi.bearer.token.provider.configuration.DefaultBearerTokenProviderConfiguration.config:com.liferay.oauth2.provider.configuration.OAuth2ProviderConfiguration.config";
+		property osgi.module.configurations = "access.token.expires.in=&quot;5&quot;${line.separator}refresh.token.expires.in=&quot;10&quot;:oauth2.expired.authorizations.afterlife.duration=&quot;5&quot;${line.separator}oauth2.expired.authorizations.check.interval=&quot;1&quot;";
+
+		ProductMenu.gotoPortlet(
+			category = "Configuration",
+			panel = "Control Panel",
+			portlet = "OAuth 2 Administration");
+
+		var clientId = "abc123";
+		var clientSecret = "abc123";
+		var portalURL = PropsUtil.get("portal.url");
+
+		OAuth2.addApplication(
+			applicationName = "Test Expired Application",
+			callbackURI = '''http://martamedio.com/oauth2-tester/?url=${portalURL}&client_id=${clientId}''');
+
+		Navigator.gotoBack();
+
+		OAuth2.editScopes(
+			applicationName = "Test Expired Application",
+			resourceCheckList = "liferay-json-web-services.everything.read.userprofile",
+			resourcePanels = "Portal Services");
+
+		LexiconEntry.gotoEntryMenuItem(
+			menuItem = "Edit",
+			rowEntry = "Test Expired Application");
+
+		OAuth2.inputApplicationValues(
+			clientId = "${clientId}",
+			clientSecret = "${clientSecret}");
+
+		var authorizationCode = OAuth2.getAuthorizationCode();
+
+		var curl = '''
+			${portalURL}/o/oauth2/token \
+				-d client_id=abc123 \
+				-d client_secret=abc123 \
+				-d grant_type=authorization_code \
+				-d code=${authorizationCode}
+		''';
+
+		echo("curl command ${curl}");
```

Comment:
@dellawang Why aren't we using code like this to do this curl post? I'm assuming this is step 7 in https://issues.liferay.com/browse/LRQA-57882?
```
var portalVersionOutput = JSONCurlUtil.post("${portalVersionCurl}");

if (!(contains("${portalVersionOutput}", "7.3"))) {
	fail("JSON-WS call failed to return Portal version");
}
```